redash/cli/users.py

```diff
@@ -84,3 +103,19 @@ def list():
             print "-" * 20
 
         print "Id: {}\nName: {}\nEmail: {}".format(user.id, user.name.encode('utf-8'), user.email)
+
+
+def build_groups(groups, is_admin):
+    org = models.Organization.get_by_slug('default')
+    if isinstance(groups, basestring):
+        groups= groups.split(',')
+        groups.remove('') # in case it was empty string
+        groups = [int(g) for g in groups]
+
+    if groups is None:
+        groups = [models.Group.get(models.Group.name=="default", models.Group.org==org).id]
```

Comment:
If we already touching this code, then you can use `org.default_group` and `org.admin_group` helper methods.